catalogs/catalog-hadoop/src/main/java/com/datastrato/gravitino/catalog/hadoop/kerberos/AuthenticationConfig.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2024 Datastrato Pvt Ltd.
+ * This software is licensed under the Apache License version 2.
+ */
+
+package com.datastrato.gravitino.catalog.hadoop.kerberos;
+
+import com.datastrato.gravitino.Config;
+import com.datastrato.gravitino.config.ConfigBuilder;
+import com.datastrato.gravitino.config.ConfigConstants;
+import com.datastrato.gravitino.config.ConfigEntry;
+import com.datastrato.gravitino.connector.PropertyEntry;
+import com.google.common.collect.ImmutableMap;
+import java.util.Map;
+
+public class AuthenticationConfig extends Config {
+  public static final String AUTH_TYPE_KEY = "hadoop.authentication.type";
```

Comment:
Maybe we should use `authentication.type`.

Reply:
This is the configuration of Hadoop catalog, why do we need to remove `hadoop` here?

Reply:
Hadoop catalog properties don't start with `hadoop`. You can refer to `location`. `hadoop` is rebudant information.

Reply:
Let me check it.